src/iotjs_binding.cpp

```diff
@@ -70,34 +70,32 @@ JObject::JObject(const JObject& other) {
 
 
 JObject::JObject(bool v) {
-  _obj_val = JVal::Bool(v);
+  _obj_val = iotjs_jval_bool(v);
```

Comment:
Well, it is true that they have same meaning, but I didn't removed that wrapper function because:
* The explicit type of `_obj_val` is `JRawValueType`, not `jerry_value_t`. 
* I think compiler can inline the wrapper function. (checked at x86_64)
* We can add more common optimization in iotjs_jval_* (e.g. returning pre-declared value instead of calling jerry api, at least until we have LTO support also in iotjs)